frontend/src/pages/SignupPage/index.js

```diff
@@ -47,38 +97,39 @@ const SignupPage = () => {
       );
       navigator("/policy");
     } else {
-      setStep((prev) => {
-        switch (prev) {
-          case STEP_1:
-            return STEP_2;
-          case STEP_2:
-            return STEP_3;
-          default: {
-          }
-        }
-      });
       setError({ hasError: false, message: "" });
```

Comment:
No message?

Reply:
Yeah, I defined as empty because Input has a default error message when there is no error